app/models/anime.rb

```diff
@@ -59,6 +59,7 @@ class Anime < ApplicationRecord
   has_many :anime_productions, dependent: :destroy
   has_many :anime_characters, dependent: :destroy
   has_many :anime_staff, dependent: :destroy
+  has_many :media_attribute, through: :anime_media_attributes
```

Comment:
Yes to the first question.
And it has at least three now, and maybe more in the future.